src/models/profileModel.ts

```diff
@@ -20,10 +30,10 @@ export interface ProfileDocument extends Document {
   photos?: string[];
```

Comment:
this has to be changed, photos will be  an object, you can look it up in the another PR in this repo